### PR TITLE
install and enablle libaom

### PR DESCRIPTION
**Pull request by TeamoreA**

### CODE REVIEW

File: nvidia-php8.1-fpm-ffmpeg-bullseye/Dockerfile [hunk 1]
```diff
@@ -60,12 +60,20 @@ FROM base as ffmpeg-builder
 RUN apt -y install build-essential cmake gcc git libc6 libc6-dev libtool nasm nvidia-cuda-toolkit yasm
 # Ffmpeg
 WORKDIR /src
+# Install libaom from source.
+RUN mkdir -p ~/ffmpeg_sources/libaom && \
+  cd ~/ffmpeg_sources/libaom && \
+  git clone https://aomedia.googlesource.com/aom && \
+  cmake ./aom && \
+  make && \
+  sudo make install
```
Comment (victortrac) [inline nvidia-php8.1-fpm-ffmpeg-bullseye/Dockerfile]: There's no reason to make a random directory like this.  PWD is already /src.  See how nv-codec-headers is built.

```
RUN git clone https://aomedia.googlesource.com/aom && cd aom && \
      cmake . && make && make install
```

**Review by victortrac — CHANGES_REQUESTED** [review]

File: nvidia-php8.1-fpm-ffmpeg-bullseye/Dockerfile (hunk 1 above)
Comment (TeamoreA) [reply nvidia-php8.1-fpm-ffmpeg-bullseye/Dockerfile]: seen that, i have updated the installion path

**Review by victortrac — APPROVED** [review]

**Review by victortrac — CHANGES_REQUESTED** [review]
please alpha sort

**Comment by TeamoreA** [comment]
> 



> please alpha sort

I have worked on this.

**Review by victortrac — APPROVED** [review]

**Pull request merged**